crt_portal/cts_forms/models.py

```diff
@@ -36,6 +36,9 @@ class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     intake_filters = models.TextField(max_length=500, blank=True)
 
+    def __str__(self):
+        return self.intake_filters
```

Comment:
This renders the contents of the `intake_filters` attribute (example below), which isn't particularly helpful since it will be modified and vary over time. I'd suggest `user` so that a developer or admin can see to which user each profile is associated.

<img width="489" alt="Screen Shot 2020-09-08 at 4 13 15 PM" src="https://user-images.githubusercontent.com/3485564/92523370-3f04a180-f1ee-11ea-807d-b743cae479cc.png">

Reply:
Would you recommend the something like self.user or self.user._str_